PeerioTechnologies/peerio-desktop
# chore: make all illustration images non draggable

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/16879/desktop-illustrations-shouldn-t-be-draggable

#### Testing instructions
Illustrations in app should not be draggable. This is most noticeable in an empty folder, where dragging the background image causes the drag-and-drop upload UI highlight to show up. But in general it's something that can happen by accident and looks weird.

Affected illustrations:
* room invite
* chat zero state
* files zero state (including empty folder)
* contact list zero state and contact invite

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
